examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
So we went ahead and released `0.12.2`, but we would like ot have a very quick follow on `0.12.3` possibly as soon as two weeks. There's several PRs like this one ready or almost ready to merge so there is no reason to wait. In any case, I think as soon as you can revert the example code we can get this merged. Thanks!